app/views/foods/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<h2>Foods</h2>
+<table class="recipe-table">
+  <thead class="table-head">
+    <tr>
+      <th>Food</th>
+      <th>Measurement unit</th>
+      <th>Unit price</th>
+      <th>Actions</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @foods.each do |food| %>
+      <tr>
+        <td><%= food.name %></td>
+        <td><%= food.measurement_unit %></td>
+        <td><%= number_to_currency(food.price) %></td>
+        <td>
+          <%= link_to 'Edit', edit_food_path(food), class: 'table-btn' %> |
+          <%= button_to 'Delete', food_path(food), method: :delete, form: { data: { confirm: 'Are you sure?' } }, class: "delete-button-foods" %>
```

Comment:
- [ ]  Let's give it our best shot to mirror the design of our foods page based on the wireframe provided. This helps maintain a consistent look and feel across our application. 🎨🔍

![image](https://github.com/hajnaloltyan/recipe-app/assets/98453979/cd764575-1d3f-4662-96a8-f058338a3a48)

#### Checkout the original wirefram

![image](https://github.com/hajnaloltyan/recipe-app/assets/98453979/fc0a1a1e-559f-4926-bd87-472a33b6bf41)

### Optional

- [ ]  I'd suggest removing the edit button since the edit action isn't present in your controllers. Let's ensure our application components are all in good working order! 🔧🚫

Reply:
- Resolved
<img width="1266" alt="Screenshot 2023-11-25 at 21 20 37" src="https://github.com/hajnaloltyan/recipe-app/assets/58692590/6b072d9b-9bb4-4680-b480-2086dca7c974">

- Great job moving the add food button to the top, however, the actions need to be placed side by side, I will level this as an optional requirement but kindly ensure to fix this so the app can look more presentable.